packages/lib/services/searchengine/SearchFilterNonLatin.test.js

```diff
@@ -0,0 +1,453 @@
+/* eslint-disable no-unused-vars, @typescript-eslint/no-unused-vars, prefer-const */
```

Comment:
I know it's copied and pasted but normally new files should be TypeScript and with no estlint-disable.